consensus/core/src/metrics.rs

```diff
@@ -37,161 +66,190 @@ pub(crate) fn test_metrics() -> Arc<Metrics> {
+                registry,
+            )
+                .unwrap(),
+            fetch_blocks_scheduler_inflight: register_int_gauge_with_registry!(
+                "fetch_blocks_scheduler_inflight",
+                "Designates whether the synchronizer scheduler task to fetch blocks is currently running",
                 registry,
             ).unwrap(),
+            fetched_blocks: register_int_counter_vec_with_registry!(
+                "fetched_blocks",
+                "Number of fetched blocks per peer authority via the synchronizer.",
+                &["authority", "type"],
+                registry,
+            ).unwrap(),
+            // TODO: add a short status label.
+            invalid_blocks: register_int_counter_vec_with_registry!(
+                "invalid_blocks",
+                "Number of invalid blocks per peer authority",
+                &["authority", "source"],
+                registry,
+            )
+                .unwrap(),
             last_committed_leader_round: register_int_gauge_with_registry!(
                 "last_committed_leader_round",
                 "The last round where a leader was committed to store and sent to commit consumer.",
                 registry,
             ).unwrap(),
-            decided_leaders_total: register_int_counter_vec_with_registry!(
-                "committed_leaders_total",
-                "Total number of (direct or indirect) committed leaders per authority",
-                &["authority", "commit_type"],
+            last_decided_leader_round: register_int_gauge_with_registry!(
+                "last_decided_leader_round",
+                "The last round where a commit decision was made.",
                 registry,
-            )
-            .unwrap(),
-            block_commit_latency: register_histogram_with_registry!(
-                "block_commit_latency",
-                "The time taken between block creation and block commit.",
+            ).unwrap(),
+            leader_timeout_total: register_int_counter_with_registry!(
+                "leader_timeout_total",
+                "Total number of leader timeouts",
                 registry,
             )
-            .unwrap(),
-            blocks_per_commit_count: register_histogram_with_registry!(
-                "blocks_per_commit_count",
-                "The number of blocks per commit.",
+                .unwrap(),
+            missing_blocks_total: register_int_gauge_with_registry!(
+                "missing_blocks_total",
+                "Total number of missing blocks",
                 registry,
+            ).unwrap(),
+            quorum_receive_latency: register_histogram_with_registry!(
+                "quorum_receive_latency",
+                "The time it took to receive a new round quorum of blocks",
+                registry
             )
             .unwrap(),
+            scope_processing_time: register_histogram_vec_with_registry!(
```

Comment:
super nit: `scoped_processing_time` seems to read slightly better, when it means the processing time of a specific scope.